ext/rotoscope/rotoscope.c

```diff
@@ -275,6 +282,7 @@ VALUE rotoscope_stop_trace(VALUE self)
   if (rb_tracepoint_enabled_p(config->tracepoint))
   {
     rb_tracepoint_disable(config->tracepoint);
+    config->state = RS_OPEN;
```

Comment:
Would be good to document this: https://github.com/Shopify/rotoscope/blob/4a2abd08bc538b9c91dcd9d853afe1e4ad8674ff/README.md#rotoscopestop_trace